spark/v3.2/spark/src/main/java/org/apache/iceberg/spark/actions/BaseExpireSnapshotsSparkAction.java

```diff
@@ -70,7 +72,7 @@
     extends BaseSparkAction<ExpireSnapshots, ExpireSnapshots.Result> implements ExpireSnapshots {
   private static final Logger LOG = LoggerFactory.getLogger(BaseExpireSnapshotsSparkAction.class);
 
-  public static final String STREAM_RESULTS = "stream-results";
+  public static final String PARALLEL_DELETE = "parallel_delete";
```

Comment:
This should not remove unrelated options.

Also, options should use `-` to separate words, not `_`.